pkg/controller/daemon/daemon_controller_test.go

```diff
@@ -1565,7 +1564,6 @@ func TestNodeShouldRunDaemonPod(t *testing.T) {
 					},
 				},
 			},
-			wantToRun:             true,
```

Comment:
nit: wantToRun in line 1552 can also be removed

Reply:
done